history-of-quakers/modernized/26-chapter-24.adoc

```diff
@@ -1380,6 +1440,10 @@ wherefore the people called Quakers esteemed it their duty
 to show also their grateful acknowledgment of this favor,
 which they did by the following address.
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
 To Queen ANNE, of Great Britain, etc.
 
 The humble and thankful Address of the Queen`'s Protestant subjects,
```

Comment:
I didn't if this should be [ssco] because it's giving a location, but it doesn't have a date.

Reply:
I guess looking at it more, I"m not sure if it's stating where the letter is coming from per se, but that it's from the Quakers who are in and about London....

Reply:
Yeah, tough one. I think I like it as a ssco, like you did with the one further down. Let's make this one too.